### ISSUE TITLE
Calling getGeofences() throws an exception

### ISSUE THREAD

**Issue by tpberntsen** [transistorsoft/flutter_background_geolocation#3]
Hi Chris,

When invoking getGeofences() like in the following example, an exception is thrown:

```
bg.BackgroundGeolocation.getGeofences().then((List<bg.Geofence> geofences) {
  // Iterate over the geofences
});
```

The exception thrown is:

```
Line 192: Exception has occurred.
type 'MappedListIterable<dynamic, Geofence>' is not a subtype of type 'FutureOr<List<Geofence>>'
```

I believe this is the line of code involved:

https://github.com/transistorsoft/flutter_background_geolocation/blob/096f16b78d0df6f9ab933e3b043c7808b75496da/lib/src/background_geolocation.dart#L192

Thanks,
Thomas

**Comment by christocracy** [comment]
Should be fixed in 193671bca0c0c472052e0f51ee37f46927af70db

```
flutter_background_geolocation:
    git:
      url: https://github.com/transistorsoft/flutter_background_geolocation
      ref: 193671bca0c0c472052e0f51ee37f46927af70db
```

**Comment by christocracy** [comment]
Fixed an iOS type error in `getGeofences'.  commit-hash above is modified to b1c4a6a8637f5365331d94ad16af838d19b925fe

**Comment by christocracy** [comment]
Fixed in `0.1.2`
